docs/fuse.md

```diff
@@ -44,7 +44,29 @@ sudo chown <username>:<groupname> /etc/fuse.conf
 
 ## Troubleshooting
 
-If the mount command crashes and your mountpoint gets stuck:
+### Getting `Permission denied` or `fusermount: user has no write access to mountpoint` error
+
+Verify that the config file can be read by your user:
+```sh
+ls -l /etc/fuse.conf
+-rw-r----- 1 root fuse 216 Jan  2  2013 /etc/fuse.conf
+```
+In most distributions group named `fuse` will be created during installation.
```

Comment:
maybe add: 

> In most distributions group named `fuse` will be created during installation. You can check with:
>
> ```
> <cross-platform command to check if fuse group exists here>
> ```

for `<cross-platform command to check if fuse group exists here>`, maybe: 

```
cat /etc/group | grep fuse
```
?